packages/dds/tree/src/feature-libraries/sequence-field/index.ts

```diff
@@ -0,0 +1,15 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export * from "./format";
+export * from "./sequenceFieldChangeHandler";
+export * from "./sequenceFieldChangeRebaser";
+export * from "./sequenceFieldChangeEncoder";
+export * from "./sequenceFieldToDelta";
+export * from "./sequenceFieldEditor";
+export * from "./markListFactory";
+export * from "./rebase";
+export * from "./invert";
+export * from "./compose";
```

Comment:
We unfortunately do for the sake of tests. This is a consequence of my choice use `TestChange` instead of `NodeChangeset` in some of the tests: tests need the definition of those function that doesn't assume `NodeChangeset` is used everywhere.